deploy/jenkins-ci/Dockerfile

```diff
@@ -62,7 +62,9 @@ RUN set -eux; \
     # Clean-up \
     rm -rf /var/lib/apt/lists/*; \
     # Set exec permissions to entrypoint; \
-    chmod +x /usr/local/bin/entrypoint.sh;
+    chmod +x /usr/local/bin/entrypoint.sh; \
+    # Do some things to support generating operator all-in-one yaml \
+    apt-get -qq update; apt-get -qq install gettext-base;
```

Comment:
Could you move these new lines before the apt clean-up (before line 63)? Just to have an image with reduced size. If you move this before the clean-up you may not need to invoke `apt-get -qq update`.

Reply:
this is done - there is now just 2 lines added to dockerfile (one is a comment) and I took out the apt-get update.